docs/eventing/samples/container-source/README.md

```diff
@@ -0,0 +1,133 @@
+ContainerSource will start a container image which will generate events under certain situations and send messages to a sink uri. It also can be an easy way to support your own event sources in Knative. This document shows how to configure ContainerSource as event source for functions and summarize guidelines for creating your own event source as ContainerSource.
```

Comment:
You need to add an _index.md file for the /container-source/ folder, which will include the title of the page.

For an example, see: https://raw.githubusercontent.com/knative/docs/master/docs/eventing/samples/cronjob-source/_index.md

Reply:
Thanks. Fixed.